crates/fluvio-benchmark/src/benches.rs

```diff
@@ -0,0 +1,41 @@
+use async_std::{task::spawn, future::timeout, prelude::FutureExt};
+use criterion::black_box;
+
+use crate::{Setup, DEFAULT_TIMEOUT};
+
+/// Spawns two async threads (one for producer and one for consumer), waits for them to complete and then joins them.
+/// Consumer thread returns when it has read all expected messages
+pub async fn run_throughput_test(setup: Setup) {
+    let (producer, consumer) = setup;
+    // producer.produce().await;
+    // consumer.consume().await;
+    let producer_jh = spawn(timeout(DEFAULT_TIMEOUT, producer.produce()));
```

Comment:
The "latency" is being measured as the time from the start of sending the first record to the end of receiving the last record.